### ISSUE TITLE
Alpine Linux 3.10 (latest stable) has several build issues w/dependencies

### ISSUE THREAD

**Issue by nmarley** [dashpay/dash#3090]
### Can you reliably reproduce the issue?

**In Alpine Linux 3.10**:

```sh
apk update
apk add git cmake make curl gcc g++ perl automake autoconf libtool pkgconfig python3

git clone -b develop https://github.com/dashpay/dash.git && cd dash
cd depends
make -j32  # or whatever number
```

### Expected behaviour

Dependencies should build without an issue.

### Actual behaviour

Build issues which prevent compilation for several packages:

- [x] openssl (known issue w/`termio` which is fixed in later versions)
- [x] chia_bls (can't find `<gmp.h>` header)
- [x] zeromq (4.1.5 patches are too fuzzy for Alpine `patch`)

### What version of Dash Core are you using?

This is on `develop` as of commit c22169d57.

### Machine specs:
- OS: Alpine Linux 3.10
- CPU/RAM: AWS m5.8xlarge
- Disk size: 80 GB
- Disk Type (HD/SDD): EBS volume

### Any extra information that might be useful in the debugging process.

If desired, for debugging on AWS, there are community Alpine AMIs offered here: <https://github.com/mcrute/alpine-ec2-ami/blob/master/releases/README.md>

**Comment by nmarley** [comment]
There is also an issue with Qt but I'll wager most people won't be wanting Alpine + Qt as the GUI is not very compatible with the goal of Alpine (small, lightweight and mostly used for servers).

**Comment by nmarley** [comment]
This is now resolved with the recently merged PRs, verified building on Alpine 3.10 using branch `develop` at commit ac0270871.
